test-cypress/steps/home-steps.js

```diff
@@ -0,0 +1,128 @@
+class HomeSteps {
+
+    static tutorialVisible(shouldBeVisible) {
+        if (shouldBeVisible) {
+            cy.get('.tutorial-container')
+                .should('be.visible');
+        } else {
+            cy.get('.tutorial-container')
+                .should('not.be.visible');
+        }
+    }
+
+    static declineTutorial() {
+        // Click "No, thanks" button
+        cy.get('.tutorial-container .decline-tutorial').click();
+    }
+
+    static selectSPARQLQueryToExecute = function (query) {
+        cy.get('ul.saved-queries li:contains("' + query + '")')
+            .should("be.visible")
+            .trigger('hover')
+            .within(() => {
+                cy.get('.execute-saved-query')
+                    .click({force: true});
+            });
+    };
+
+    static verifyQueryLink(queryName, modifiesRepoModal, queryURL) {
+        HomeSteps.selectSPARQLQueryToExecute(queryName);
+        modifiesRepoModal ? cy.get('.modal-body').should('be.visible') : cy.get('.modal-body').should('not.be.visible');
+        cy.url().should('eq', Cypress.config("baseUrl") + queryURL);
+        cy.visitAndWaitLoader('/');
+    }
+
+    static selectRepo(repositoryId) {
+        cy.visitAndWaitLoader('/');
+        cy.get('ul.repos')
+            .contains(repositoryId)
+            .should('be.visible')
+            .click();
+    }
+
+    static createRepo() {
+        let repositoryId = 'home-repository-' + Date.now();
+        cy.createRepository({id: repositoryId});
+        return repositoryId;
+    }
+
+    static isSelectedRepository(repositoryId) {
+        cy.get('ul.repos')
+            .contains(repositoryId).and('be.hidden');
+        cy.get('#btnReposGroup')
+            .should('contain', repositoryId);
+    }
+
+    static hasRepositoryInfo(repositoryId) {
+        cy.get('.active-repo-card').should('contain', repositoryId)
+            .and('contain', 'total statements')
+            .and('contain', 'explicit')
+            .and('contain', 'inferred')
+            .and('contain', 'expansion ratio')
+            .within(() => {
+                cy.get('.total-statements').should('contain', '70');
+                cy.get('.explicit-statements').should('contain', '0')
+                cy.get('.inferred-statements').should('contain', '70');
+            });
+    }
+
+    static isCorrectCreateRepositoryLink() {
+        cy.get('.card.repository-errors').should("be.visible")
+            .within(() => {
+                cy.get('.create-repository-btn')
+                    .click()
+                    .url()
+                    .should('eq', Cypress.config("baseUrl") + '/repository/create?previous=home');
+            });
+        cy.get('.big-logo').click();
+    }
+
+    static isCorrectTutorialText = function(number, text) {
```

Comment:
`verifyTutorialText` ?

Reply:
At some point I got annoyed at using this verify word everywhere, but okay, I can keep it.